bin/oscapd-evaluate

```diff
@@ -245,9 +244,9 @@ def cli_scan(args, config):
         sys.stderr.write("Evaluation interrupted by user!\n")
 
     if len(failed_targets) > 0:
-        sys.stderr.write(
+        logging.info(
             "Fatal error encountered while evaluating! Failed to evaluate "
-            "at least %i targets!\n" % (len(failed_targets) - 1)
+            "at least %i targets!\n" % len(failed_targets)
         )
```

Comment:
It is not clear to me why there was `len(failed_targets) - 1`. Although it says "at least x", which makes the message to be correct, why not to report the right number of failiures?

Reply:
Because if you scan multi-jobs in parallel you may not see everything that failed at this moment. There is a good reason for this IIRC.

Reply:
OK, I will revert this back.

Reply:
Although the reasoning is strange - if I may not see everything that failed, then "at least `len(failed_targets)`" would make sense, there is no reason why to decrement count of failed targets that we know of already.